app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,70 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
```

Comment:
@lfu You also need to override the `default_retirement_entry_point` so you do not pick up the setting from the base class.  See [app/models/service_template.rb#L302-L304](https://github.com/ManageIQ/manageiq/blob/5edac5566820d4d4b14bf55bdcc75b1daca693e5/app/models/service_template.rb#L302-L304).  You want to return `nil` in this case.